.eslintrc.js

```diff
@@ -308,6 +309,10 @@ module.exports = {
         /**
          * react
          */
+        "react/no-string-refs": "off", // on in react/recommended, but we have #legacy-code
+        "react/no-find-dom-node": "off", // on in react/recommended, but we have #legacy-code
+        "react/display-name": "off", // on in react/recommended, but doesn't seem that useful to fix
+        "react/no-unescaped-entities": "off", // on in react/recommended, but we have #legacy-code
```

Comment:
It seems like unescaped HTML entities should be easy to fix, though it also seems low-value to fix them if they're in code that we're not actively working on.

Reply:
They were easy to fix. I enabled this rule, took 20 min and fixed them all. Thanks for the nudge. :)